### PR TITLE
add support for managing ILS instance

### PR DESCRIPTION
**Pull request by anikachurilova**
* add possibility to use ILS flavour in  'invenio-cli init' command
* specify cookiecutter-invenio-ils link for ILS flavour
* closes https://github.com/inveniosoftware/invenio-cli/issues/309

### CODE REVIEW

File: invenio_cli/commands/local.py [hunk 1]
```diff
@@ -46,22 +47,24 @@ def _copy_statics_and_assets(self):
...
-
-        assets = "assets"
-        src_dir = self.cli_config.get_project_dir() / assets
-        src_dir = str(src_dir)
-        dst_dir = self.cli_config.get_instance_path() / assets
-        dst_dir = str(dst_dir)
-        # The full path to the files that were copied is returned
-        copied_files = copy_tree(src_dir, dst_dir)
-        return copied_files
+        if os.path.isdir("static"):
```
Comment (ntarocco) [inline invenio_cli/commands/local.py:50]: Why this check `isdir` is needed?

File: invenio_cli/helpers/cookiecutter_wrapper.py [hunk 2]
```diff
@@ -68,6 +68,30 @@ def __init__(self, flavour, **kwargs):
...
 
+        if self.flavour.upper() == "ILS":
+            self.template = (
+                self.template_name
+                or "https://github.com/inveniosoftware/cookiecutter-invenio-ils.git"
+            )
+            self.template_name = self.extract_template_name(self.template)
+            self.checkout = self.checkout or "v1.0rc1"
+
+        if self.flavour.upper() == "ILS":
```
Comment (ntarocco) [inline invenio_cli/helpers/cookiecutter_wrapper.py:79]: Something might be wrong here, the code is duplicated 3 times :)

File: invenio_cli/commands/local.py (hunk 1 above)
Comment (anikachurilova) [reply invenio_cli/commands/local.py:50]: We will have to run the `invenio-cli install `command, which will perform this copying of non-existing in ILS static and assets folders

Comment (ntarocco) [inline invenio_cli/commands/local.py:50]:
```suggestion
        # such folders do not exists in non-RDM contexts
        rdm_static_dir_exists = os.exists(...) and os...isidir(...)
        rdm_assets_dir_exists...
        
        if rdm_static_dir_exists:
```

**Review by ntarocco — APPROVED** [review]

**Pull request merged**